### ISSUE TITLE
Error messages ought to be in Yoruba

### ISSUE THREAD

**Issue by toystars** [anoniscoding/yorlang#21]
This is a cool project and would be fun to see it grow, and maybe the language might one day be rewritten with itself.

But I think error messages should be printed in Yoruba, or maybe run program with a flag that specifies which language results, warnings and error messages should be displayed in.

**Comment by skyusuf15** [comment]
I just saw this project and already willing to contribute to the project, I would love to help out in this section.

**Comment by AliAkolade** [comment]
I'd also love to join in on this.

**Comment by timoty-toby** [comment]
I'm so happy for this, I'm also gonna help where needed

**Comment by skyusuf15** [comment]
I totally agree with you @anoniscoding, I would suggest yorlang to have a config where you can specify or change error/warning message's language and also all error/warning messages should be in a separate file like you have for `constants`, export and import it in any other file that wants to display error/warning messages. This way you can easily change messages across the whole project and the implementation of multi-language thing would be very easy

**Comment by anoniscoding** [comment]
i agree @skyusuf15 . Looking forward to a PR from you guys

**Comment by MeNsaaH** [comment]
How do non-Yorubas come into this?

**Comment by FeezyHendrix** [comment]
@MeNsaaH well this language is an esoteric language with its focus on yoruba as its core concept .... I think, but I also feel a more inept explanation should go into the docs.

**Comment by FeezyHendrix** [comment]
@MeNsaaH yes as it would seem, I dont know if @anoniscoding has any mastermind solution to this ?
